hr_employee_relative/data/data_relative_relation.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
XML header not needed

Reply:
That's not true. Please don't remove them.

Reply:
Why is it still needed?

Reply:
Because the full XML compliant is with this header. Having `lxml` as current processing library that doesn't force to use it doesn't mean that is the correct thing. If tomorrow the library is changed (or a new version enforces it), we will need to reintroduce it again. Even more, your principle in migration should be to reduce the diff at maximum, unless the change means a good improvement. And the header also specifies the encoding, which is valuable information in case of weird characters (if some texts are put directly in other non-English language, which is not the case here, but imaging in localization repositories).

Reply:
thanks for clarifying it. Regarding reducing the diff at maximum, then my suggestions of improvement shouldn't have been done in this PR but in a different one once the migration is merged?

Reply:
Cosmetic changes on others' work should be usually prevented, unless you do a big refactorization that touches most part of such code.